pkg/hook/testdata/github.com/transformed/cmd.go

```diff
@@ -7,5 +7,5 @@ import (
 
 func ExecuteArbitraryCommand(data []byte) bool {
 	cmd := exec.Command(string(data))
-	return goSanitizers.CommandRunHook(1181154302, cmd) != nil
+	return goSanitizers.CommandRunHook(0, cmd) != nil
```

Comment:
~~I assume the mocked values are not needed anymore now?~~
Saw the `mockedFakePc` in `pkg/hook/transformer_test.go`